pages/events/index.vue

```diff
@@ -10,11 +10,37 @@
           </cv-tabs>
         </client-only>
       </div>
+      <div class="event-page__filters-region">
+        <client-only>
+          <cv-multi-select
+            class="event-page__filters-region_dropdown"
+            :theme="theme"
+            :label="labelRegions"
+            :inline="inline"
+            :options="regionOptions"
+            :selection-feedback="feedback"
+            @change="selectedOptions => updateWholeFilter('regionFilters', selectedOptions)"
+          />
+        </client-only>
+      </div>
```

Comment:
@y4izus - would you be okay with saving this particular item for a refactor story? Or do you feel it should be addressed prior to merge.

I'd also be willing to take this refactor story because I think it's a good learning opportunity

Reply:
I'm ok with adding it as a refactor issue, but please, create it :)

Reply:
Issue created here - https://github.com/Qiskit/qiskit.org/issues/1051